deploy/kubernetes/base/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ resources:
 - node.yaml
 - controller.yaml
 - setup-cluster.yaml
+- rbac_add_snapshotter.yaml
```

Comment:
Can you append this to "setup-cluster.yaml" instead? This is where all the rbacs usually live.

Reply:
Ack